src/main/webapp/assets/js/config.js

```diff
@@ -100,3 +100,17 @@ require([
 ], function (Router) {
     new Router().startBrooklynGui();
 });
+
+/*
+ * Prepend a base URL to REST API calls
+ */
+$.ajaxSetup({
+    beforeSend: function(jqXHR, settings) {
+//        var baseURL = "/api/brooklyn/";
+        var baseURL = "";
+
+        if (baseURL && settings.url.match(/^\/v1\/\w+/i)){
```

Comment:
@sliceratwork i don't think so:

```
% node
> '/api/v1/project'.indexOf('/v1/')
4
```

btw we extend the prototype of `String` in `brooklyn-utils.js` so you could just write  `.startsWith('/v1/')`.